### ISSUE TITLE
PageUp and PageDown not working as shortcuts

### ISSUE THREAD

**Issue by Ernits** [Liresol/anki-custom-shortcuts#45]
Hi, I saw a previous (solved) issue where use of arrow keys and page up/down were fixed. However, they still do not work for me. I use Anki 2.1.15. Any ideas what I'm doing wrong?

```
{
    "editor _extras": {
        "paste custom text": "<nop>"
    },
    "editor add media": "F3",
    "editor bold": "Ctrl+B",
    "editor card layout": "Ctrl+L",
    "editor change col": "F8",
    "editor cloze": "Ctrl+Shift+C",
    "editor cloze alt": "Ctrl+Shift+Alt+C",
    "editor focus tags": "Ctrl+Shift+T",
    "editor foreground": "F7",
    "editor html edit": "Ctrl+Shift+X",
    "editor insert latex": "Ctrl+T, T",
    "editor insert latex equation": "Ctrl+T, E",
    "editor insert latex math environment": "Ctrl+T, M",
    "editor insert mathjax block": "Ctrl+M, E",
    "editor insert mathjax chemistry": "Ctrl+M, C",
    "editor insert mathjax inline": "Ctrl+M, M",
    "editor italic": "Ctrl+I",
    "editor record sound": "F5",
    "editor remove format": "Ctrl+R",
    "editor subscript": "Ctrl+=",
    "editor superscript": "Ctrl++",
    "editor underline": "Ctrl+U",
    "m_toolbox _duplicates": {},
    "m_toolbox addons": "Ctrl+Shift+A",
    "m_toolbox create filtered deck": "F",
    "m_toolbox export": "Ctrl+E",
    "m_toolbox import": "Ctrl+Shift+I",
    "m_toolbox preferences": "Ctrl+P",
    "m_toolbox quit": "Ctrl+Q",
    "m_toolbox see documentation": "F1",
    "m_toolbox study": "/",
    "m_toolbox switch profile": "Ctrl+Shift+P",
    "m_toolbox undo": "Ctrl+Alt+Z",
    "main add": "A",
    "main browse": "B",
    "main debug": "Ctrl+:",
    "main deckbrowser": "D",
    "main stats": "T",
    "main study": "S",
    "main sync": "Y",
    "reviewer _duplicates": {},
    "reviewer bury card": "-",
    "reviewer bury note": "=",
    "reviewer choice 1": "1",
    "reviewer choice 2": "2",
    "reviewer choice 3": "3",
    "reviewer choice 4": "4",
    "reviewer delete note": "Ctrl+Delete",
    "reviewer edit current": "E",
    "reviewer flip card 1": ".",
    "reviewer flip card 2": "Qt.Key_PageDown",
    "reviewer flip card 3": " ",
    "reviewer mark card": "*",
    "reviewer options menu": "o",
    "reviewer play recorded voice": "v",
    "reviewer record voice": "Shift+v",
    "reviewer replay audio 1": "r",
    "reviewer replay audio 2": "F5",
    "reviewer set flag 0": "Ctrl+0, Qt.Key_PageUp",
    "reviewer set flag 1": "Ctrl+1",
    "reviewer set flag 2": "Ctrl+2",
    "reviewer set flag 3": "Ctrl+3",
    "reviewer set flag 4": "Ctrl+4",
    "reviewer suspend card": "@",
    "reviewer suspend note": "!",
    "window_browser add note": "Ctrl+E",
    "window_browser add tag": "Ctrl+Shift+A",
    "window_browser change deck": "Ctrl+D",
    "window_browser change note type": "Ctrl+Shift+M",
    "window_browser clear unused tags": "<nop>",
    "window_browser close": "Ctrl+W",
    "window_browser delete": "Ctrl+Del",
    "window_browser filter": "Ctrl+Shift+F",
    "window_browser find": "Ctrl+F",
    "window_browser find and replace": "Ctrl+Alt+F",
    "window_browser find duplicates": "<nop>",
    "window_browser first card": "Home",
    "window_browser flag_blue": "Ctrl+4",
    "window_browser flag_green": "Ctrl+3",
    "window_browser flag_orange": "Ctrl+2",
    "window_browser flag_red": "Ctrl+1",
    "window_browser goto card list": "Ctrl+Shift+L",
    "window_browser goto next note": "Ctrl+N",
    "window_browser goto note": "Ctrl+Shift+N",
    "window_browser goto previous note": "Ctrl+P",
    "window_browser goto sidebar": "Ctrl+Shift+R",
    "window_browser guide": "F1",
    "window_browser info": "Ctrl+Shift+I",
    "window_browser invert selection": "Ctrl+Alt+S",
    "window_browser last card": "End",
    "window_browser manage note types": "<nop>",
    "window_browser preview": "Ctrl+Shift+P",
    "window_browser remove tag": "Ctrl+Shift+D",
    "window_browser reposition": "Ctrl+Shift+S",
    "window_browser reschedule": "Ctrl+Alt+R",
    "window_browser select all": "Ctrl+Alt+A",
    "window_browser select notes": "<nop>",
    "window_browser suspend": "Ctrl+J",
    "window_browser toggle mark": "Ctrl+K",
    "window_browser undo": "Ctrl+Z",
    "Ω custom paste end style": "n",
    "Ω custom paste text": "",
    "Ω enable conflict warning": "y"
}
```

**Comment by Liresol** [comment]
This might actually be a combination of a few weird coincidences.
Right now, Anki only has 4 flags, so `"reviewer set flag 0"` will actually do nothing regardless of shortcut.
As a note, the assignment of two shortcuts to one function for the reviewer is currently handled by the `"reviewer _duplicates"` object. 

There was an actual issue in the program itself, for which I have just pushed out an update. 

If there are any further issues, let me know and I will further investigate.

**Comment by Ernits** [comment]
Hi, you've fixed it! 
Also, after I posted, I realized I didn't want to change hotkey to flag 1, but to choice 1. 
Idea was, I have a Logitech R400 presenter remote and I wanted to use it for Anki. However, it turns out the next and previous slide buttons are equivalent to hitting Page Down and Page Up on the keyboard, respectively. It also has a button for black screen, which is a "." 

So I've gone with the following solution, which now works perfectly. Thanks!

```
 "reviewer _duplicates": {
        "reviewer choice 1": "Qt.Key_PageUp",
        "reviewer choice 4": "."
    },
"reviewer flip card 2": "Qt.Key_PageDown",
```
